src/main/resources/injection.st

```diff
@@ -0,0 +1,16 @@
+import org.springframework.util.AntPathMatcher
+import org.springframework.util.PathMatcher
+
+PathMatcher antMatcher = new AntPathMatcher("\0")
```

Comment:
Or better: as it could be considered a standard feature in Groovy contingency list DSL, we can maybe try to simplify the matching to users by directly expose a function match(str1, str2) to script so with no need to create a AntPathMatcher

Reply:
Done